frontend/packages/kubevirt-plugin/integration-tests/tests/models/virtualMachine.ts

```diff
@@ -148,7 +221,7 @@ export class VirtualMachine extends KubevirtDetailView {
         // Rootdisk is present by default, only edit specific properties
         await wizard.editDiskAttribute(WIZARD_TABLE_FIRST_ROW, 'size', resource.size);
         await wizard.editDiskAttribute(WIZARD_TABLE_FIRST_ROW, 'storage', resource.storageClass);
-      } else if (resource.attached === true) {
+      } else if (resource.source === DISK_SOURCE.AttachDisk) {
```

Comment:
there should be only one method addDisk which will behave according to the source.

Reply:
Yep, fixed